target/linux/ramips/mt7620/base-files/etc/board.d/01_leds

```diff
@@ -195,6 +195,19 @@ tplink,archer-mr200)
 tplink,re200-v1)
 	ucidef_set_led_netdev "lan" "lan" "green:lan" "eth0"
 	;;
+wavlink,wl-wn579x3)
+	ucidef_set_led_switch "lan" "lan" "blue:lan" "switch0" "0x2f"
+	ucidef_set_led_switch "wan" "wan" "blue:wan" "switch0" "0x10"
+	# associate the 'wps' led with wan0, the 5GHz interface
+	ucidef_set_led_netdev "wifi0_led" "wifi0" "blue:wps" "wlan0"
+	# associate the 'wifi' led with wan1, the 2GHz interface
+	ucidef_set_led_netdev "wifi1_led" "wifi1" "blue:wifi" "wlan1"
```

Comment:
Nope. Unless the vendor does this, too, please leave such custom settings to the user.

Reply:
Thanks. I need to check what the default behaviour is if i don't do anything here, I have a feeling that the WIFI LED had some default behaviour.

From the user's perspective the WIFI LED lights up when at least one of the two wlan's is connected; (assuming I misremembered and there is no hardware default) do you know if it is possible to associate both wlan0/1 to the same LED here?

If there is not, what would you suggest? It feels like a better out-of-box experience to associate it with at least one of the wlan interfaces, than neither?

Reply:
In any case, for WiFi activity a phy trigger in DTS is preferred.

It's not possible to have two radios linked to one LED by default. Since choice is not pre-determined here, in that case the choice should be left to the user, i.e. no trigger is defined.

For the rssileds, the same argument about the number of interfaces applies.

Reply:
I consider an arbitrary assignment worse. If no LED is active, the user will have a trigger to take care himself. If there is LED activity, the user will assume it works "correctly" and has to find out his assumptions are wrong the hard way.

Reply:
Ok, no problem, I'll take that and the rssi stuff out and leave it to the end user.